admin/code/CampaignAdmin.php

```diff
@@ -62,6 +64,10 @@ public function getClientConfig() {
 			],
 			'campaignViewRoute' => $urlSegment . '/:type?/:id?/:view?',
 			'itemListViewEndpoint' => $this->Link('set/:id/show'),
+			'publishEndpoint' => [
+				'url' => $this->Link('set/:id/publish'),
```

Comment:
Be careful; Link takes an action, and in this case only `set` is a valid action. Additional arguments should probably be appended afterwards.